web/modules/custom/farm_nfa/src/Form/ForestPlanGfwForm.php

```diff
@@ -27,9 +27,13 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     // Set the form title.
     $form['#title'] = $this->t('GFW');
 
-    $form['placeholder'] = [
-      '#type' => 'markup',
-      '#markup' => '(GFW form placeholder)',
+    $form['gfw_map_placeholder'] = [
+      '#type' => 'farm_map',
+      '#map_type' => 'farm_nfa_plan_locations',
+      '#map_settings' => [
+        'plan' => \Drupal::routeMatch()->getRawParameter('plan')
```

Comment:
Please check what "dependency injection" is and inject the route match instead, it's a better practice